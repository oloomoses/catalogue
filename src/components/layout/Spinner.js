/* eslint-disable no-use-before-define */
import React from 'react';
import spinner from './spinner.gif';

const Spinner = () => (
  <div>
    <img src={spinner} alt="loading ..." style={spinnerStyle} />
  </div>
);

const spinnerStyle = {
  width: '200px',
  margin: 'auto',
  display: 'block',
};

export default Spinner;
