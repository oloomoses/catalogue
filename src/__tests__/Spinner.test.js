import renderer from 'react-test-renderer';
import Spinner from '../components/layout/Spinner';

describe('renders correctly', () => {
  it('matches the snapshot', () => {
    const tree = renderer.create(<Spinner />);
    expect(tree).toMatchSnapshot();
  });
});
