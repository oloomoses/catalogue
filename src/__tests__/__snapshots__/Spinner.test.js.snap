// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly matches the snapshot 1`] = `
<div
  className="container text-center"
>
  <img
    alt="loading ..."
    src="spinner.gif"
  />
</div>
`;
